qlib/data/updateparallel.py

```diff
@@ -0,0 +1,35 @@
+from joblib import Parallel
+
+
+class UpdateParallel(Parallel):
```

Comment:
`UpdateParallel` moves to `qlib/utils/__init__.py`
`UpdateParllel` renamed to `ParallelExt`

Reply:
`https://github.com/microsoft/qlib/blob/main/qlib/utils/paral.py` will be a better place